components/GitHubAccelerator.js

```diff
@@ -35,25 +35,25 @@ export default function GitHubAccelerator() {
             <div className="mt-6 flex flex-col justify-stretch space-y-3 sm:flex-row sm:space-x-4 sm:space-y-0">
               <Link
                 href="http://accelerator.github.com"
-                className="inline-flex justify-center rounded-md bg-white px-3 py-2 text-sm font-semibold text-primary-high dark:text-primary-high shadow-sm ring-1 ring-inset ring-primary-low-medium hover:bg-primary-low dark:hover:bg-slate-200 cursor-pointer"
+                className="inline-flex ml-1.7 justify-center rounded-md bg-white px-3 py-2 text-sm font-semibold text-primary-high dark:text-primary-high shadow-sm ring-1 ring-inset ring-primary-low-medium hover:bg-primary-low dark:hover:bg-slate-200 cursor-pointer"
```

Comment:
We should not try to sign items with margins, because as the length of the content changes so will the alignment requirements

It is best use alignment, at the moment it is `justify-center` but for smaller screens I think from your screenshot, left align would solve this

Reply:
Thank you for clarifying this, I've removed the margin alignment (ml-1.7 ).